tools/deployment/presto-clp/scripts/init.py

```diff
@@ -274,6 +280,45 @@ def _generate_worker_clp_properties(
     return True
 
 
+def _get_clp_package_instance_id(
+    clp_config: Dict[str, Any], clp_package_dir: Path
+) -> Optional[str]:
+    """
+    Retrieves the CLP package instance ID from the logs directory.
+
+    :param clp_config:
+    :param clp_package_dir:
+    :return: The instance ID if it could be read, otherwise ``None``.
+    """
+
+    logs_directory = _get_path_clp_config_value(
+        clp_config, "logs_directory", Path("var") / "log", clp_package_dir
+    )
+    instance_id_path = logs_directory / "instance-id"
+    if not instance_id_path.exists():
+        logger.error(
+            "Cannot determine the CLP package Docker network because '%s' does not exist. "
+            "Start the CLP package at least once before configuring Presto.",
```

Comment:
```suggestion
            "Cannot determine the CLP package Docker network because '%s' does not exist."
            " Start the CLP package at least once before configuring Presto.",
```